gping/src/colors.rs

```diff
@@ -28,14 +27,23 @@ where
 
     fn next(&mut self) -> Option<Self::Item> {
         match self.color_names.next() {
-            Some(name) => match try_color_from_string(name) {
+            // TODO: As of v0.21.0, Ratatui only accepts color patterns
+            // formatted as "lightred" or "light red". We replace any '-' in
+            // the color name with a space for compatibility.
+            //
+            // Note: Revisit this section when Ratatui supports patterns such
+            // as "light-red". At that point, this replacement operation
+            // will be unnecessary and can be removed.
```

Comment:
Could you add a link to https://github.com/tui-rs-revival/ratatui/issues/305 here?

Reply:
Sure, thank you for you quick response!